src/veins/modules/utility/TimerManager.h

```diff
@@ -27,6 +27,13 @@
 
 namespace Veins {
 
+/**
+ * A class for managing and timers.
```

Comment:
How about something more concrete, like 
```suggestion
 * Abstraction for (recurring) Timers for cObject to get rid of self-mesages.
```
Maybe even more motivation on why one should use this and some example.